stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/info/ChannelInfo.kt

```diff
@@ -83,42 +82,48 @@ public fun ChannelInfo(
         }
```

Comment:
A user can leave any channel with client generated id (non-distinct channel, not starting with "!members")
A user can delete any channel that they own, also admins can delete any channel.

Reply:
Sounds good!